UnityProject/Assets/Editor/LevelBuilder/GUI/Menus/GridMenu.cs

```diff
@@ -8,43 +8,39 @@
  * Allows the user to choose when to display the grid and how big it should be.
  */
 
-public class GridMenu : IMenu
-{
+public class GridMenu: IMenu {
 	private bool showGrid = true;
 
 	private string width = "10";
 	private string height = "10";
 
-	public void Display()
-	{
-		EditorGUILayout.LabelField ("", GUI.skin.horizontalSlider);
+	public void Display() {
+		EditorGUILayout.LabelField("", GUI.skin.horizontalSlider);
 
+		// Enclosing in a BeginHorizontal() to indicate when to end and
+		// move to the next horizontal line.
 		GUILayout.BeginHorizontal();
 
 		// Grid toggle.
 		showGrid = EditorGUILayout.Toggle("Grid", showGrid);
 
-		// Width and height labels and text fields.
-		GUILayout.Label("Width: ");
-		width = GUILayout.TextField(width, GUILayout.MaxWidth(30));
-		GUILayout.Label("Height: ");
-		height = GUILayout.TextField(height, GUILayout.MaxWidth(30));
+		// Move to next line in the GUI.
+		GUILayout.EndHorizontal();
+		GUILayout.BeginHorizontal();
+
+		width = GUIUtils.TextField("Width: ", width);
+		height = GUIUtils.TextField("Height: ", height);
 
+		// Move to next line in the GUI.
 		GUILayout.EndHorizontal();
 
-		if (showGrid) 
-			try 
-			{
-				GridVisualiser.BuildGrid (Convert.ToInt32 (width), Convert.ToInt32 (height));
-			}
-			catch (FormatException exception) 
-			{
-				Debug.Log (
-					"Grid width/height format exception." +
-					"Please make sure you write an integer.");
-			}
-		else
-			GridVisualiser.DestroyGrid();
+		if (showGrid) try {
+			GridVisualiser.BuildGrid(Convert.ToInt32(width), Convert.ToInt32(height));
+		}
+		catch(FormatException exception) {
+			Debug.Log("Grid width/height format exception." + "Please make sure you write an integer.");
```

Comment:
Not sure teenagers will know what an integer is!

Reply:
The unity level builder will only be used by level designers, changed anyways!

Reply:
That's fine then, but we need to be more descriptive when talking about "users" I think

Reply:
Definitely:) I'll keep that in mind